lib/utils.js

```diff
@@ -24,21 +24,38 @@ function constructObject(object, keys) {
 
 function calculateDeadline(
 	roundDeadlines = [
-		'Sep 18 2022',
-		'Nov 20 2022',
-		'Jan 29 2023',
-		'Mar 19 2023',
-		'Apr 16 2023',
+		'Sep 03 2023',
+		'Oct 15 2023',
+		'Dec 03 2023',
+		'Feb 04 2024',
+		'Mar 17 2024',
+		'Apr 07 2024',
 	],
-	outcomeDates = ['Sep 28 2022', 'Nov 30 2022', 'Feb 08 2023', 'Mar 29 2023','Apr 26 2023']
+	outcomeDates = [
+		'Sep 13 2023',
+		'Oct 25 2023',
+		'Dec 13 2023',
+		'Feb 14 2024',
+		'Mar 27 2024',
+		'Apr 17 2024',
+	]
 ) {
 	let visualDeadline,
 		technicalDeadline,
 		outcome,
 		currentDate = moment().tz('Asia/Kolkata')
 
 	let round = 1
-	const ordinals = ['', 'First', 'Second', 'Third', 'Fourth', 'Final']
+	const ordinals = [
+		'',
+		'First',
+		'Second',
+		'Third',
+		'Fourth',
+		'Fifth',
+		'Sixth',
+		'Final',
```

Comment:
If there are 6 deadlines then the sixth one will be the final one, so in this case the 'Final' text will never be shown. If you want the sixth one to be the final one then remove 'Sixth', otherwise remove 'Final'

Reply:
makes sense.. We have 6 rounds now :

Round 1	Sunday, 3rd September 2023
Round 2	Sunday, 15th October 2023
Round 3	Sunday, 3rd December 2023
Round 4	Sunday, 4th February 2024
Round 5	Sunday, 17th March 2024
Round 6	Sunday, 7th April 2024